docs/user/installconfig.schema.json

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
There should be one file for the generic properties, and two (which now that I think of it, can probably be condensed in one) per platform.

All could also fit in one big file, but I thought that by splitting them, each platform team could have agency on their part.

Since it's probably going to be one file per platform (+1 generic root), I would tend to keep them close to the relative docs, so that a developer modifying the docs _might_ notice that they also could update the schema. WDYT?

Reply:
I have pushed a change in which all platform-specific properties are condensed in one file.